test/com/gfredericks/test_schpec.clj

```diff
@@ -41,3 +41,10 @@
   (is (nil? (s/explain-data ::my-xor {::y 1})))
   (is (some? (s/explain-data ::my-xor {::x 1 ::y 1})))
   (is (some? (s/explain-data ::my-xor {::y "abc"}))))
+
+(schpec/alias 'p 'person)
+
+(deftest alias-test
+  (let [aliases (ns-aliases 'com.gfredericks.test-schpec)]
```

Comment:
this makes me wonder if it's possible to register an alias at a lower level without having to create the namespace? That might be good for tools that enumerate through all the namespaces and might get confused